test/WebJobs.Script.Tests.Integration/Host/StandbyManagerTests.cs

```diff
@@ -32,74 +31,66 @@ namespace Microsoft.Azure.WebJobs.Script.Tests
+            Assert.True(environment.IsContainerReady());
 
-                // give time for the specialization to happen
-                string[] logLines = null;
-                await TestHelpers.Await(() =>
-                {
-                    // wait for the trace indicating that the host has been specialized
-                    logLines = _loggerProvider.GetAllLogMessages().Where(p => p.FormattedMessage != null).Select(p => p.FormattedMessage).ToArray();
-                    return logLines.Contains("Generating 0 job function(s)");
-                }, userMessageCallback: () => string.Join(Environment.NewLine, _loggerProvider.GetAllLogMessages().Select(p => $"[{p.Timestamp.ToString("HH:mm:ss.fff")}] {p.FormattedMessage}")));
-
-                // verify the rest of the expected logs
-                Assert.True(logLines.Count(p => p.Contains("Stopping Host")) >= 1);
-                Assert.Equal(1, logLines.Count(p => p.Contains("Creating StandbyMode placeholder function directory")));
-                Assert.Equal(1, logLines.Count(p => p.Contains("StandbyMode placeholder function directory created")));
-                Assert.Equal(2, logLines.Count(p => p.Contains("Starting Host (HostId=placeholder-host")));
-                Assert.Equal(2, logLines.Count(p => p.Contains("Host is in standby mode")));
-                Assert.Equal(2, logLines.Count(p => p.Contains("Executed 'Functions.WarmUp' (Succeeded")));
-                Assert.Equal(1, logLines.Count(p => p.Contains("Starting host specialization")));
-                Assert.Equal(1, logLines.Count(p => p.Contains($"Starting Host (HostId={_expectedHostId}")));
-                Assert.Contains("Generating 0 job function(s)", logLines);
-            }
+            // give time for the specialization to happen
+            string[] logLines = null;
+            await TestHelpers.Await(() =>
+            {
+                // wait for the trace indicating that the host has been specialized
+                logLines = _loggerProvider.GetAllLogMessages().Where(p => p.FormattedMessage != null).Select(p => p.FormattedMessage).ToArray();
+                return logLines.Contains("Generating 0 job function(s)") && logLines.Contains("Stopping JobHost");
+            }, userMessageCallback: () => string.Join(Environment.NewLine, _loggerProvider.GetAllLogMessages().Select(p => $"[{p.Timestamp.ToString("HH:mm:ss.fff")}] {p.FormattedMessage}")));
+
+            _httpServer.Dispose();
+            _httpClient.Dispose();
+
+            // verify the rest of the expected logs
+            Assert.True(logLines.Count(p => p.Contains("Stopping JobHost")) >= 1);
+            Assert.Equal(1, logLines.Count(p => p.Contains("Creating StandbyMode placeholder function directory")));
+            Assert.Equal(1, logLines.Count(p => p.Contains("StandbyMode placeholder function directory created")));
+            //Assert.Equal(2, logLines.Count(p => p.Contains("Starting Host (HostId=placeholder-host")));
```

Comment:
We have an issue with pulling the hostname from host.json in placeholders. @fabiocav was investigating a fix. This test will fail and need to be adjusted once that is in.

Reply:
Should we have an issue to track?

Reply:
I was just using the placeholder ID as an indicator that we started twice in placeholder mode. I'd be fine with just removing this check if the ID is no longer special or specified in host.json. We have other tests below that verify the sam e thing (e.g. 2 X "Host is in standby mode")

Reply:
Okay, that makes sense. I'll pull this.